pkg/proto/dispatch/v1/dispatch_vtproto.pb.go

```diff
@@ -0,0 +1,4497 @@
+	if err := x.ClientStream.RecvMsg(m); err != nil {
+		return nil, err
+	}
+	return m, nil
+}
+
+// DispatchServiceServer is the server API for DispatchService service.
+// All implementations must embed UnimplementedDispatchServiceServer
+// for forward compatibility
+type DispatchServiceServer interface {
+	DispatchCheck(context.Context, *DispatchCheckRequest) (*DispatchCheckResponse, error)
+	DispatchExpand(context.Context, *DispatchExpandRequest) (*DispatchExpandResponse, error)
+	DispatchLookup(context.Context, *DispatchLookupRequest) (*DispatchLookupResponse, error)
+	DispatchReachableResources(*DispatchReachableResourcesRequest, DispatchService_DispatchReachableResourcesServer) error
+	DispatchLookupSubjects(*DispatchLookupSubjectsRequest, DispatchService_DispatchLookupSubjectsServer) error
+	mustEmbedUnimplementedDispatchServiceServer()
+}
+
+// UnimplementedDispatchServiceServer must be embedded to have forward compatible implementations.
+type UnimplementedDispatchServiceServer struct {
+}
+
+func (UnimplementedDispatchServiceServer) DispatchCheck(context.Context, *DispatchCheckRequest) (*DispatchCheckResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method DispatchCheck not implemented")
+}
+func (UnimplementedDispatchServiceServer) DispatchExpand(context.Context, *DispatchExpandRequest) (*DispatchExpandResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method DispatchExpand not implemented")
+}
+func (UnimplementedDispatchServiceServer) DispatchLookup(context.Context, *DispatchLookupRequest) (*DispatchLookupResponse, error) {
+	return nil, status.Errorf(codes.Unimplemented, "method DispatchLookup not implemented")
+}
+func (UnimplementedDispatchServiceServer) DispatchReachableResources(*DispatchReachableResourcesRequest, DispatchService_DispatchReachableResourcesServer) error {
+	return status.Errorf(codes.Unimplemented, "method DispatchReachableResources not implemented")
+}
+func (UnimplementedDispatchServiceServer) DispatchLookupSubjects(*DispatchLookupSubjectsRequest, DispatchService_DispatchLookupSubjectsServer) error {
+	return status.Errorf(codes.Unimplemented, "method DispatchLookupSubjects not implemented")
+}
+func (UnimplementedDispatchServiceServer) mustEmbedUnimplementedDispatchServiceServer() {}
+
+// UnsafeDispatchServiceServer may be embedded to opt out of forward compatibility for this service.
+// Use of this interface is not recommended, as added methods to DispatchServiceServer will
+// result in compilation errors.
+type UnsafeDispatchServiceServer interface {
+	mustEmbedUnimplementedDispatchServiceServer()
+}
+
+func RegisterDispatchServiceServer(s grpc.ServiceRegistrar, srv DispatchServiceServer) {
+	s.RegisterService(&DispatchService_ServiceDesc, srv)
+}
+
+func _DispatchService_DispatchCheck_Handler(srv interface{}, ctx context.Context, dec func(interface{}) error, interceptor grpc.UnaryServerInterceptor) (interface{}, error) {
+	in := DispatchCheckRequestFromVTPool()
```

Comment:
vtprotobuf replaces our usage of the grpc plugin and generates these handlers that grab messages from the pool.